Tests/Security/TwoFactor/Provider/TwoFactorProviderRegistryTest.php

```diff
@@ -23,11 +23,20 @@ class TwoFactorProviderRegistryTest extends TestCase
      */
     private $registry;
 
+    /**
+     * @var \PHPUnit_Framework_MockObject_MockObject
+     */
+    private $eventDispatcher;
+
     public function setUp()
     {
+        $this->eventDispatcher = $this->getMockBuilder('Symfony\Component\EventDispatcher\EventDispatcher')
+            ->disableOriginalConstructor()
+            ->getMock();
```

Comment:
`Scheb\TwoFactorBundle\Tests\TestCase` provides a compatibility layer for older versions of PHPUnit, that's why createMock can be used here.